README.md

```diff
@@ -191,3 +191,20 @@ refunds = refunds.previous_page
 payment = Justifi::Payment.get(payment_id: 'py_xyz')
 refund = Justifi::Refund.get(refund_id: 're_xyz')
 ```
+
+## Seller Account
+
+You can make requests using the `Seller-Account` header in order to process resources as a seller-account.
+
+Configuration
+
+```ruby
+Justifi.seller_account = 'seller_account_public_id'
```

Comment:
One concern with the class varilable/config: Only concern with this is it is could cause issues with thread safety versus just having it passed in as a header. Maybe we go more with a block driven approach in the future? Since it will be common for those using this option to be having long living processes which process transactions potentially across sellers, we could do like this?

```
Justifi.with_seller_account('acc_id') do
 # ... the request
end
```

Be good to get some other feedback.

Reply:
Cool, It's decided then.